lib/vanagon/project.rb

```diff
@@ -270,22 +270,75 @@ def has_provides?
       !get_provides.empty?
     end
 
-    # Collects the preinstall packaging actions for the project and it's components
-    # for the specified packaging state
+    # Collects the preinstall packaging actions for the project and its components
+    #   for the specified packaging state
```

Comment:
*super nitpick, so feel free to ignore, but* this space is not really necessary at all